semialign/src/Data/Align.hs

```diff
@@ -6,6 +6,7 @@
 -- functors see "Data.Crosswalk".
 module Data.Align (
     Semialign (..),
+    Semizip (..),
```

Comment:
No I don't think we should re-export `Semizip` from `Data.Align`:

- `Data.Align` has the `Align` part of the story
- `Data.Zip` has the `Zip` part of the story (and `Semialign` for necessity)
- `Data.Semialign` is badly named, but is "everything". (I'll think what its name should be.)

Reply:
Fair enough - just a tradeoff between how much we break things and how much we make things make sense.  I'll update it.